app/ProjectAanvraag/Listeners/CreateWidget.php

```diff
@@ -139,4 +140,12 @@ private function handle(UuidInterface $integrationId): void
             )
         );
     }
+
+    public function failed(ContactCreated $contactCreated, Throwable $throwable): void
```

Comment:
This is a bit tricky because it listens to 3 different events, it can fail on all of these 3 events. Maybe a union type works?

Reply:
Here you go:

[Show correct logs per entity type](https://github.com/cultuurnet/publiq-platform/pull/906/commits/32ce9447944e6b5e13581419a6ba4ca6b2ba5816)